### PR TITLE
feat: implementing pyarrow round test

### PR DESCRIPTION
**Pull request by brentomagic**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [ ] ✅ Test
- [ ] 🐳 Other

## Related issues 

- Related issue # 
- Closes #

## Checklist

- [ ] Code follows style guide (ruff)
- [ ] Tests added 
- [ ] Documented the changes

### CODE REVIEW

File: narwhals/_arrow/expr.py [hunk 1]
```diff
@@ -165,6 +165,11 @@ def std(self, ddof: int = 1) -> Self:
     def cast(self, dtype: DType) -> Self:
         return reuse_series_implementation(self, "cast", dtype)
 
+    def round(self, decimals: int) -> Self:
+        return reuse_series_implementation(
+            self, "round", decimals=decimals, returns_scalar=True
```
Comment (MarcoGorelli) [inline narwhals/_arrow/expr.py:170]: does this return a scalar?

File: narwhals/_arrow/series.py [hunk 2]
```diff
@@ -221,6 +221,10 @@ def n_unique(self) -> int:
         count_unique = pc.count(unique_values, mode="all")
         return count_unique.as_py()  # type: ignore[no-any-return]
 
+    def round(self, decimals: int = 0) -> Self:
+        pc = get_pyarrow_compute()
+        return pc.round(self._native_series, ndigits=decimals)  # type: ignore[no-any-return]
```
Comment (EdAbati) [inline narwhals/_arrow/series.py:226]: here we are returning a `pyarrow.ChunkedArray`, instead we need a `ArrowSeries` (as the typing righly says)

**Comment by DeaMariaLeon** [comment]
@brentomagic if the title of the PR begins with any of this: (build|chore|ci|depr|docs|feat|fix|perf|refactor|release|test), the PR goes to the right place on the release document. 
I'll correct your title so you see what I mean. 🙂

**Comment by FBruzzesi** [comment]
I am closing this in favor of #696

**Pull request closed without merging**